Source/ACE/Entity/Player.cs

```diff
@@ -2746,5 +2753,35 @@ public void DoMoveTo(WorldObject wo)
 
             moveToObjectChain.EnqueueChain();
         }
+
+        public bool CheckMagicSkillLevel(MagicSchool school, uint power)
+        {
+            bool ret = false;
+            CreatureSkill creatureSkill = Skills[Skill.Run];
```

Comment:
poor coding is at play here.. i needed to define creatureSkill but the switch changes it to the right thing, vstudio complained it was unassigned. It shouldn't be possible to ever actually stay as run, but there probably a smarter way to do what I'm doing